module_auto_update/migrations/9.0.2.0.0/pre-migrate.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Tecnativa - Jairo Llopis
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+import openupgradelib
+
+from openerp.addons.module_auto_update.models.module_deprecated import \
+    PARAM_DEPRECATED
+
+
+# TODO Remove when deprecated features are removed; see README's known issues
+@openupgradelib.migrate(no_version=True, use_env=True)
+def migrate(env):
+    """Move stored checksums to new JSON format."""
+    # Store module checksums in the new JSON format
+    env.cr.execute("""SELECT name, checksum_installed
+                      FROM ir_module_module
+                      WHERE checksum_installed""")
+    checksums = dict(env.cr.fetchall())
+    env["ir.module.module"]._save_checksums(checksums)
```

Comment:
note the checksum algorithm as changed so this not help.

Reply:
You are correct, I'll do a couple of upgrade tests just to be sure and remove it, thanks :wink: